rusk-recovery/src/state/snapshot/wrapper.rs

```diff
@@ -0,0 +1,67 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use std::ops::Deref;
+
+use dusk_bytes::{BadLength, InvalidChar, ParseHexStr};
+use dusk_bytes::{DeserializableSlice, Serializable};
+use serde::{Deserialize, Serialize};
+
+#[derive(PartialEq, Eq)]
+pub(super) struct Wrapper<C, const N: usize>(C)
```

Comment:
I'm not really sure why this structure needs to exist. Could you explain?

Reply:
As explained, it's to handle the serialisation of those type inside nested structs (eg: Vec, Hashset, etc)